packages/components/src/FormField/FormField.tsx

```diff
@@ -281,11 +283,32 @@ export function FormField({
       disabled: disabled,
       readOnly: readonly,
       onChange: handleChange,
-      value: value,
+      value:
+        typeof name !== "undefined" && typeof value !== "undefined"
+          ? watch(name)
+          : undefined,
       inputMode: keyboard,
-      ...(defaultValue && { defaultValue: defaultValue }),
+      ...(defaultValue
+        ? { defaultValue: defaultValue }
+        : value && { defaultValue: value?.toString() }),
```

Comment:
This is forcing a `defaultValue` and removing the `value` on all inputs. Is that intended?

Reply:
I have removed this and updated the `watch` function to use the `defaultValue` as a default value if specified or `value`. This fixes issues with first render not displaying values on the `<Select/>` component.